unfetter-discover/unfetter-ui
# Issue 1144/baseline stats

Fixes unfetter-discover/unfetter#1144

Totals for groups and attack patterns are now in place.

### To test:
1. Pull this branch
2. Create a new baseline and make a note of how many groups, capabilities, and attack patterns are represented in the baseline.
3. Once saved, the 3 totals should be accurately reflected on the summary screen.
